catalog/package.json

```diff
@@ -41,7 +41,8 @@
     "test:clean": "rimraf ./coverage",
     "test": "cross-env NODE_ENV=test jest --coverage",
     "test:watch": "cross-env NODE_ENV=test jest --watchAll",
-    "coveralls": "cat ./coverage/lcov.info | coveralls"
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "config": "sh -c 'env $(grep -v '^#' $0 | xargs) envsubst < config.js.tmpl > config.js'"
```

Comment:
do we really need to use `grep` to filter out comments? bash will ignore them anyway if we need it the regex for comments is more like `'^\s*#'`.

Reply:
this line does not work as written (I'm on Mac); when fixed please document in catalog/README.md

Reply:
ok, now it's using bash parsing (via `source`).

> this line does not work as written (I'm on Mac)

@akarve could you please provide your input / output? i'm on mac also, but it works for me